### ISSUE TITLE
Should we cite the WCAG 2.2 "Accessible Authentication" proposal? If so, what should be the approach taken?

### ISSUE THREAD

**Issue by jasonjgw** [w3c/captcha-accessibility#11]
This issue was raised by John Rochford on the APA Working Group mailing list.
The thread starts at
https://lists.w3.org/Archives/Public/public-apa/2021Dec/0031.html
